code/script.js

```diff
@@ -0,0 +1,81 @@
+const projectCard = document.getElementById('projectCard')
+const profile = document.getElementById('profile')
```

Comment:
Best practice is to minimize declaring variables in the global scope. I noticed that all three of these are not necessary in the global scope, so they make most sense being moved in to the scope where they're used. I'll add additional comments as an example.